sbws/core/scanner.py

```diff
@@ -571,5 +571,4 @@ def main(args, conf):
     # Generate an unique identifier for each scanner
     if 'uuid' not in state:
         state['uuid'] = str(uuid.uuid4())
-
```

Comment:
This is one more extraneous minor format change.

Reply:
ok, fixup to solve that